go.mod

```diff
@@ -156,7 +156,7 @@ require (
 	github.com/Masterminds/semver v1.5.0 // indirect
 	github.com/Masterminds/semver/v3 v3.1.1 // indirect
 	github.com/Masterminds/sprig v2.22.0+incompatible // indirect
-	github.com/Masterminds/sprig/v3 v3.2.1 // indirect
+	github.com/Masterminds/sprig/v3 v3.2.1
```

Comment:
Is this tidy? I think Go may want to pull this to the block above?

Reply:
Tidying made no change to the code 🤷